Entities/CatEntityTest.php

```diff
@@ -0,0 +1 @@
+<?php
```

Comment:
Why is this unused file in this PR?

Reply:
all the files that will be used in the project were created at the very beginning stage for everyone's convenience

Reply:
Remove this file, too.